pkgs/development/python-modules/flake8-polyfill/default.nix

```
@@ -11,6 +11,16 @@ buildPythonPackage rec {
     sha256 = "1nlf1mkqw856vi6782qcglqhaacb23khk9wkcgn55npnjxshhjz4";
   };
 
+  patches = [
+    # Notified upstream. Check if this is merged before the next release.
+    # TODO: Update this patch when there is a version upgrade to flake8.
```

```suggestion
    # allow newer flake8 versions
    # update when there is a version upgrade to flake8.
```

The patch will fail to apply if it is in the release in this form.